tests/deps/setup_rejson.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Exit immediately with statys 1 (error) if a command exits with a non-zero status
+set -e
+
+# Set the default variables
+ROOT=${ROOT:=`pwd`}
```

Comment:
There are probably better ways to get the root directory of the project. I don't think this will work. Consider fetching the file location and going two steps back

Reply:
We are passing ROOT to the script when we run tests, this default initialization is for benchmarks where this script is called directly from GitHub actions (and it works) - other ways just look more backward no?

Reply:
I think assuming the caller to the script is at the root directory is a bad practice in writing a script. IMO it is much more portable to assume the relative path of the file in the repo and use it to find the root directory, than enforcing the caller to be at the right directory when calling it, especially when the script is not at the root directory